### DISCUSSION TITLE
Práctica Js

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#434] [Q&A]
Buenas noches apenas empezaré la práctica de JS. 

El código de JS se tiene que hacer con los progamas de JS o se puede hacer igual con VUE JS.

¿Qué recomiendan en general para realizar la práctica?

**Comment by MauroMontan** [comment]
Si no tienes conocimiento en vue es mejor realizar la práctica en javascript para que practiques más, antes de empezar con vue

**Reply by GloriaCoralCerecedo** [reply]
Eso mismo pensé para practicar cada cosa
